core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -1,5 +1,5 @@
 # numaresources-operator
-registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator quay.io/openshift-kni/numaresources-operator:4.10-snapshot
+registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator quay.io/openshift-kni/numaresources-operator:4.10.0
```

Comment:
we will have a problem here when we move to the Z stream, in that case the ".0" part will be misleading. Let's call it `4.10.0-snapshot` at least

Reply:
the more I think about it, the more it seems the closest semanti version value I can think of is `4.10.999-snapshot`. This should be explicit enough to suggest that it is a random Y-stream snapshot, and not necessarily a 4.10.0 variant.

Reply:
I got to admit that this seems a bit cumbersome.
IMHO ```4.10.0-snapshot``` is ok, i don't expect confusion with Z streams, especially if you're looking at the existing patterns in the file which have a single mirror for every Y-stream.

Reply:
I agree `4.10.999-snapshot` is far from ideal. It seems to me however that having a `4.10.0-snapshot` image with content referring to, say, `4.10.12` is not great either.

Reply:
OK. let me change it

Reply:
This will also require a change in NROP Makefile for the default ```VERSION``` value, but let's merge this one first.